src/v-sticky/sticky.vue

```diff
@@ -0,0 +1,26 @@
+<script>
+
+    export default {
+        inserted (el, binding) {
+
+            const offset = binding.arg
+
+            window.addEventListener('scroll', () => {
+
+                if (!el.getBoundingClientRect) return
```

Comment:
Make sure `getBoundingClientRect` is a function - this is just checking for the existence of the `getBoundingClientRect` property on `el`, which will probably always return true for an element